.github/workflows/unit-tests.yml

```diff
@@ -79,7 +79,7 @@ jobs:
 
       - name: Set up WordPress
         run: |
-          bash <(curl -s "https://raw.githubusercontent.com/alleyinteractive/mantle-ci/HEAD/install-wp-tests.sh") wordpress_test root '' 127.0.0.1 ${{ matrix.wp_version }}
+          bash <(curl -s "https://raw.githubusercontent.com/alleyinteractive/mantle-ci/HEAD/install-wp-tests.sh") wordpress_unit_tests root '' 127.0.0.1 ${{ matrix.wp_version }} false true
```

Comment:
It would be nice if this action could use our reusable https://github.com/alleyinteractive/.github/blob/main/.github/workflows/php-tests.yml workflow, although that may not be possible with the customizations we need to make in order to install Elasticsearch. However, since Mantle is able to perform the installation steps, we could rely on it to do that work instead of manually running install-wp-tests, like our reusable action does. This could also be an enhancement we add later via a new issue instead of needing to be handled as part of this PR since it represents a change from how we set it up initially.

Reply:
I will add this enhancement issue now!